OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -350,6 +349,18 @@ public void Nudge(Actor self, Actor nudger, bool force)
 			}
 		}
 
+		public static bool Ismovable(Actor otherActor)
```

Comment:
Both for readability and consistency the 'm' should be upper case, too, since this isn't really a single word.

Reply:
Fixed.